### PR TITLE
Task/google cloud deploy

### PR DESCRIPTION
**Pull request by vijayyadav06**
#### Move Service locator UI project from Heroku to GCP. #20
1. Adding app.yaml for GCP app engine install
2. Added cloudbuild.yaml for using CloudBuild and its trigger for continuous deployment
3. Added Content Security Policy headers

### CODE REVIEW

**Comment by vijayyadav06** [comment]
Can be accessed here - https://scode-test.appspot.com/, still need to purchase a domain name.

File: server.js [hunk 1]
```diff
@@ -7,6 +7,20 @@ const distPath = path.resolve(`${__dirname}/dist`);
 const port = process.env.PORT || 8080;
 const app = express();
 
+let sources = {
+  'default-src': ["'self'"],
```
Comment (mihiramin89) [inline server.js]: why is the double quotes necessary?

File: package.json [hunk 2]
```diff
@@ -46,7 +48,8 @@
     "router5-plugin-browser": "^7.0.1",
     "showdown": "^1.9.0",
     "throttle-debounce": "^2.0.1",
-    "universal-ga": "^1.2.0"
+    "universal-ga": "^1.2.0",
+    "webpack": "^4.28.2"
```
Comment (mihiramin89) [inline package.json]: Do we need to remove this as a devDependency?

**Comment by mihiramin89** [comment]
We should unlink the heroku connection when this is merged.

File: package.json (hunk 2 above)
Comment (vijayyadav06) [reply package.json]: nopes, still needed when running locally in dev mode. When we deploy to GCP, prod being the default devDependencies are ignored.

File: server.js (hunk 1 above)
Comment (vijayyadav06) [reply server.js]: its mandatory to have single quotes around the policy. https://developers.google.com/web/fundamentals/security/csp/ (search for 'These keywords require single-quotes').

**Pull request merged**